go-controller/pkg/util/multi_network.go

```diff
@@ -111,6 +112,7 @@ type NetConfInfo interface {
 	TopologyType() string
 	MTU() int
 	Subnets() []string
+	IPMode() (bool, bool)
```

Comment:
this name does not hint much at something that returns a pair of bools - I would expect it to return the supported modes, not indication of if v4 / v6 are supported. 

Maybe `IsIPv4V6Supported` ? ... It's ugly, but hints at returning booleans. 

Let's see if someone else has better suggestions.

Reply:
the existing config variable is config.IPv4Mode and config.IPv6Mode. So I followed the convention. We'd need to know both.